docs/oauth2-client.md

```diff
@@ -0,0 +1,83 @@
+Oauth2 client setup
+===================
+
+Setting up auth
+---------------
+
+To get the API up and running, you first need to have [a12n-server][1] installed.
+Installation instructions can be found [here][2].
+
+After a12nserver is running, and you confirmed you can log in, you need to
+create separate new user & database. (Follow [a12n-server][1] installation for
+ `MySQL setup` section)
+
+ ```sh
+mysql> CREATE DATABASE my-app-db;
+mysql> CREATE USER 'my-app' IDENTIFIED BY 'your_password';
+mysql> GRANT SELECT, INSERT, UPDATE, DELETE ON my-app-db.* TO 'my-app';
+mysql> FLUSH PRIVILEGES;
+```
```

Comment:
I don't think these instructions belong here. a12nserver is a generic OAuth2 server, and it doesn't make any assumptions about the language/database/application that is talking to it.